### ISSUE TITLE
How to install in parallel both stable and nightly kitty program?

### ISSUE THREAD

**Issue by sheeeng** [nix-community/home-manager#6071]
How to install latest nightly kitty program?

Also, is it possible to install in parallel both stable and nightly kitty programs?

https://sw.kovidgoyal.net/kitty/binary/#customizing-the-installation

**Comment by stelcodes** [comment]
Something like this should work if the build steps haven't changed:
```nix
{
  nixpkgs.overlays = [
    (self: super: {
      kitty-nightly = super.kitty.overrideAttrs (previousAttrs: {
        src = super.fetchFromGitHub {
          owner = "kovidgoyal";
          repo = "kitty";
          rev = "<rev>";
          hash = "<hash>";
        };
      });
    })
  ];
}
```

**Comment by teto** [comment]
once you put them in PATH, the executable names will collide. You will need to patch the derivation to rename the executable. Anyway it's out of scope for this repo.
